### PR TITLE
Added iCal and GCal export. Closes #23

### LINKED ISSUE #23: Export to iCal with proper weekly repetition and semester duration

### PR DESCRIPTION
**Pull request by G-ram**
Closes #23

### CODE REVIEW

**Comment by G-ram** [comment]
If we add the four hour offset, the ical generator will work. I just tested it on my iCal and it worked (did not work on the particular calendar we tested during Cookies because the calendar was set in California time).

**Comment by G-ram** [comment]
Will send another commit tomorrow.

**Comment by G-ram** [comment]
You can go ahead and accept the pull request. It was my computer being stupid, not the program. I will eventually clean up the code and movie it into a directive.

**Comment by natebrennand** [comment]
So this works for me testing locally w/ gcal.
@brishin, did you want to edit the names so they were not all caps?

File: src/js/models/Schedule.coffee [hunk 1]
```diff
@@ -204,6 +206,73 @@ angular.module('Courses.models')
...
+              weekDay[subsectionEvent.meetsOn[0]][0],
+              weekDay[subsectionEvent.meetsOn[0]][1],
+              parseInt(endTime[0]),
+              Math.round(parseFloat("0."+endTime[1])*60),0),
+            SUMMARY: scheduleEvent.title,
+            LOCATION: subsectionEvent.building+" "+subsectionEvent.room,
+            RRULE: "FREQ=WEEKLY;UNTIL="+ICSFormatDate(new Date(endDate[2],endDate[0],endDate[1],11,59,59))
+          })
+
+      calendar.download("ADISchedule")
```
Comment (natebrennand) [inline src/js/models/Schedule.coffee:261]: Can we rename this to `Fall_2014_Schedule`?

**Comment by brishin** [comment]
Yes, but will do post merge.

**Pull request merged**